app/assets/v2/js/pages/increase_bounty.js

```diff
@@ -36,7 +39,7 @@ $(document).ready(function() {
   var denomSelect = $('select[name=denomination]');
 
   localStorage['tokenAddress'] = denomSelect.data('tokenAddress');
-  localStorage['amount'] = 0;
+  localStorage['amount'] = 0.01;
```

Comment:
What does this change do?

Reply:
This ensures that the min amount is set to 0.01 as opposed to 0. 
I originally updated on the html tag but the value here over-writes that, so had to udpate this too

In Increase funding amount shouldn't be set to 0 during submit.
If the amount is defaulted 0 -> the fee banner doesn't make sense cause it would say we charge 10% fee which is 0 :P 

we set default amount to > 0 so that we don't show 10% fee which equates to 0.